Main.html

```diff
@@ -7,12 +7,14 @@
 
   <div id="container">
     <div id="menu_container">
-      <ul id="menu">
-        <li><a href="#container">Главная</a></li>
-        <li><a href="#rez">Резюме</a></li>
-        <li><a href="#cont">Контакты</a></li>
-        <li><a href="">Продам гараж</a></li>
-      </ul>
+      <nav>
+        <ul id="menu">
+          <li><a href="#container">Главная</a></li>
+          <li><a href="#rez">Резюме</a></li>
+          <li><a href="#cont">Контакты</a></li>
+          <li><a href="">Продам гараж</a></li>
+        </ul>
+      </nav>
     </div>
     <div id="avatar_container">
       <div id="photo"></div>
```

Comment:
I don't think that in this case it is reasonable to use **background-image** for your avatar. Please, use **img** element with **src** and **alt** attribute.

Reply:
fixed